en/sensor/sf45_rotating_lidar.md

```diff
@@ -0,0 +1,76 @@
+
+The [parameters to change](../advanced_config/parameters.md) are listed in the table.
+
+| Parameter                                                                                                   | Description                                              |
+| ----------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| <a id="SENS_EN_SF45_CFG"></a>[SENS_EN_SF45_CFG](../advanced_config/parameter_reference.md#SENS_EN_SF45_CFG) | Set to the serial port you have the sensor connected to. |
+| <a id="SF45_ORIENT_CFG"></a>[SF45_ORIENT_CFG](../advanced_config/parameter_reference.md#SF45_ORIENT_CFG)    | Set the orientation of the sensor (facing up or down)    |
+| <a id="SF45_UPDATE_CFG"></a>[SF45_UPDATE_CFG](../advanced_config/parameter_reference.md#SF45_UPDATE_CFG)    | Set the update rate                                      |
+| <a id="SF45_YAW_CFG"></a>[SF45_YAW_CFG](../advanced_config/parameter_reference.md#SF45_YAW_CFG)             | Set the yaw orientation                                  |
+
+## Testing
+
+You can confirm that the sensor is correctly configured by connecting QGroundControl, and observing that [OBSTACLE_DISTANCE](https://mavlink.io/en/messages/common.html#OBSTACLE_DISTANCE) is present in the [MAVLink Inspector](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/analyze_view/mavlink_inspector.html).
+
+The obstacle overlay in QGC will look like this:
+
+![sf45 obstacle avoidance map shown in QGC](../../assets/hardware/sensors/lidar_lightware/sf45_obstacle_map.png)
+
+## Driver Implementation
+
+The [sensor driver](../modules/modules_driver_distance_sensor.md#lightware-sf45-serial) publishes the [ObstacleDistance](../msg_docs/ObstacleDistance.md) UORB Message that is used by PX4 [Collision Prevention](../computer_vision/collision_prevention.md).
+The measurements in each sector will correspond to the lowest measurement the sensor had in that corresponding sector.
+The data is then published to the [OBSTACLE_DISTANCE](https://mavlink.io/en/messages/common.html#OBSTACLE_DISTANCE) MAVLink message.
+
+## Debugging / Common problems
+If you are experiencing start-up issues, jerky movements or a lot of com errors displayed with `lightware_sf45_serial status` this could mean that you are having power issues.
+
+The Sensor seems to be quite sensitive in regard to whether he gets enough power.
+According to the Datasheet, the sensor needs 300 mA of current at 5V, whereby the stock cable is fairly long and can cause problems where the voltage drops too much.
+So it's important to make sure that the cable is kept as short as possible.
+As an alternative, you can power the SF45 via a separate Step-down converter from battery voltage. With this approach, you're able to eliminate all power-related issues.
```

Comment:
```suggestion
## Debugging / Common problems

Start-up issues, jerky movements, or a lot of com errors displayed with `lightware_sf45_serial status` could mean that the sensor is not getting enough power.

According to its datasheet, the sensor needs 300 mA of current at 5V.
The stock cable is fairly long and can cause problems where the voltage drops too much, so it is important to keep the cable is kept as short as possible.

One mitigation/alternative is to power the SF45 via a separate step-down converter from battery voltage.
With this approach, you're able to eliminate all power-related issues.
```

Reply:
@Claudio-Chies Where you say 

> The stock cable is fairly long and can cause problems where the voltage drops too much, so it is important to keep the cable is kept as short as possible.

Do you mean this is supplied with a cable but it is too long? Are you then proposing that the user gets a different cable that is shorter, or cuts this one back?

Depending on your answer I'll modify the suggestion above.

Reply:
Yes, the stock cable is 35cm, when using that cable, then supplying power from an BEC is recommended.

Reply:
OK, I have modified the https://github.com/PX4/PX4-user_guide/pull/3448#discussion_r1868572214 and will merge. Please check it is right now.

Reply:
i've added a commit where i specify that we are talking about the 5V supply from TELEM ports, and not any 5V supply